lib/eventFetch.js

```diff
@@ -27,31 +25,39 @@ function mapLinkList(linkList) {
   return [];
 }
 
+export function mapAndSanitateEvent(event) {
```

Comment:
Not a biggie, but I wonder if this might read a little easier if the map functions handled the condition? Either that or, as all of them seem to check for the "value" key and offer a default value we could use a predicate function or something. Nitpicking though :)